libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1746,7 +1746,7 @@ void RCOutput::serial_bit_irq(void)
             irq.bitmask = 0;
         }
     } else {
-        systime_t dt = now - irq.byte_start_tick;
+        uint16_t dt = now - irq.byte_start_tick;
```

Comment:
What happens in the timeout condition? I have seen this happen before - do we overflow?

Reply:
wrap is 65ms, which is over 120 bytes at 19200 baud